Subject: [PATCH] Added field %f to the Exec key in desktop file.
---
Index: misc/io.github.punesemu.puNES.desktop.in
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/misc/io.github.punesemu.puNES.desktop.in b/misc/io.github.punesemu.puNES.desktop.in
--- a/misc/io.github.punesemu.puNES.desktop.in	(revision 6ce00537876262957b55939645b402bd6111791c)
+++ b/misc/io.github.punesemu.puNES.desktop.in	(revision 3eed1878baaac7bdab3087fdb7299003b52faf15)
@@ -2,7 +2,7 @@
 Name=puNES
 GenericName=Nintendo Entertainment System emulator
 Comment=Nintendo Entertainment System emulator
-Exec=punes
+Exec=punes %f
 Icon=io.github.punesemu.puNES
 Type=Application
 Categories=Game;Emulator;
